prusa_slicer_post_processing_script.py

```diff
@@ -478,10 +492,15 @@ def extract_features(self)->None:
                 if currenttype:
                     self.features.append([currenttype,buff,start])
                     buff=[]
-                    start=idl                        
+                    start=idl
+
+                    if self.lines[start - 1].startswith("G1 E") or not "E" in self.lines[start - 1]: # if a travel move came right before this feature:
+                        start -= 1
+                        while not "E" in self.lines[start - 1]: # include this travel move in the feature (excluding the first retraction step, if applicable)
+                            start -= 1
```

Comment:
This is part of the fix for #56. Including the first retraction step also removes the stringing but gets rid of the retraction and de-retraction steps. Therefore, we leave out the first retraction step.